code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -346,6 +346,7 @@ impl oracle::Config for Runtime {
 	type StakeLock = StakeLock;
 	type StalePrice = StalePrice;
 	type AddOracle = EnsureRootOrHalfNativeCouncil;
+	type SetSigner = EnsureRootOrHalfNativeCouncil;
```

Comment:
EnsureRootOrHalfNativeCouncil likekely cannot be signer. 

you may consider use same pattern as on Cosmwasm or Ibc pallets. Use one of tech commette in PROD and permissionless in TEST(rococo)